Parser/asdl.py

```diff
@@ -64,34 +64,45 @@ def __init__(self, name, fields=None):
     def __repr__(self):
         return 'Constructor({0.name}, {0.fields})'.format(self)
 
+class Quantifier:
+    class _Optional: pass
+    class _Sequence: pass
+    OPTIONAL = _Optional()
+    SEQUENCE = _Sequence()
```

Comment:
Could this just be an enum?

Reply:
Sure, I just prefer these to have unique/singleton values (as opposed to `int`s or whatnot).

I can change it if its outside the expected coding style.

Reply:
If you wanted singletons you could also just do `OPTIONAL = object()` but to be honest I think an Enum/StrEnum is a better choice here since it's more explicit.

Reply:
(one reason to do the class trick is to have a bit of a nicer repr(), but OTOH, I don't think we need to worry about importing `enum`)

Reply:
Is it worth importing `enum`? If you want to add your suggested change I can commit it.

Reply:
Yes let's use an enum here, it's the right tool.